card/mod/05_standard/set/all/links.rb

```diff
@@ -5,6 +5,8 @@
   # link is called by web_link, card_link, and view_link
   # (and is overridden in other formats)
   def link_to text, href, opts={}
+    href = interpret_href href
```

Comment:
this isn't deeply related; I just noticed this inconsistency while planning my approach.